components/concept/engine/src/main/java/mozilla/components/concept/engine/request/RequestInterceptor.kt

```diff
@@ -74,7 +74,8 @@ interface RequestInterceptor {
         hasUserGesture: Boolean,
         isSameDomain: Boolean,
         isRedirect: Boolean,
-        isDirectNavigation: Boolean
+        isDirectNavigation: Boolean,
+        isSubframeRequest: Boolean
```

Comment:
OK, only request is here is to update the KDocs :) to add the new `isSubframeRequest` but also make it clearer what the difference is between `hasUserGesture`, `isDirectNavigation` and `isSubframeRequest`. I don't think I even know :).

Reply:
My mistake.  I'll add the parameter to the KDocs. 👍